pkg/standalone/testdata/test_run_config.yaml

```diff
@@ -0,0 +1,26 @@
+
+version: 1
+common:
+  resources_dir: ./app/resources
+  env:
+    - name: DEBUG
+      value: false
+    - name: tty
+      value: sts
+apps:
+  - app_id: webapp
```

Comment:
Can we have an example with app_id as optional and being inferred by the folder's name?

Reply:
yeah.. we can infer `app_id`( if it is empty) from `app-dir`. Will make the change